flux-macros/src/lib.rs

```diff
@@ -46,6 +48,28 @@ decl_derive!(
 );
 
 #[proc_macro]
-pub fn fluent_messages(input: TokenStream) -> TokenStream {
+pub fn fluent_messages(input: proc_macro::TokenStream) -> proc_macro::TokenStream {
     diagnostics::fluent_messages(input)
 }
+
+// TODO: Add extern_spec macro
+// #[proc_macro_attribute]
+// pub fn extern_spec(attr: proc_macro::TokenStream, tokens: proc_macro::TokenStream) -> proc_macro::TokenStream {
+//     transform_extern_spec(attr.into(), tokens.into())
+//         .unwrap_or_else(|err| err.to_compile_error().into())
+//         .into()
+// }
+//
+// fn transform_extern_spec(attr: TokenStream, tokens: TokenStream) -> syn::Result<TokenStream> {
+//     let fn_path: syn::Path = syn::parse2(attr)?;
+//     let item: syn::Item = syn::parse2(tokens)?;
+//     match item {
+//         syn::Item::Verbatim(stub_tokens) => {
+//
+//         }
+//         _ => {
+//             Err(syn::Error::new(Span::call_site(), "Extern specs only may be used on function stubs"))
+//         }
+//     }
+//
+// }
```

Comment:
We need to create a new macro crate. This one is for internal development and it shouldn't be exposed to users.

Reply:
Also, this new macro crate should eventually be published in crates.io to make it easy to use with `cargo-flux`